Testing/VelaptorTests/Guards/EnsureThatTests.cs

```diff
@@ -0,0 +1,57 @@
+﻿// <copyright file="EnsureThatTests.cs" company="KinsonDigital">
+// Copyright (c) KinsonDigital. All rights reserved.
+// </copyright>
+
+namespace VelaptorTests.Guards
+{
+    using System;
+    using Velaptor.Guards;
+    using VelaptorTests.Helpers;
+    using Xunit;
+
+    /// <summary>
+    /// Tests the <see cref="EnsureThat"/> class.
+    /// </summary>
+    public class EnsureThatTests
+    {
+        #region Method Tests
+        [Fact]
+        public void ArgIsNotNull_WithNullValue_ThrowsException()
```

Comment:
Rename the first section of this unit test method to match the method under test